modules/automation/automation-account/README.md

```diff
@@ -945,3 +891,61 @@ module automationAccount './automation/automation-account/main.bicep' = {
 
 </details>
 <p>
+
+
+## Notes
+
+### Parameter Usage: `encryption`
```

Comment:
Please remove the entire parameter usage guidance. The entire parameter is wrong and vastly outdated. Further, by now we have an encr deployment test that shows how to deploy CMK

Reply:
removed. Regenerated readme